src/aggregator/server/m3msg/server.go

```diff
@@ -52,12 +55,19 @@ func NewServer(
 	s := &server{
 		aggregator: aggregator,
 		logger:     opts.InstrumentOptions().Logger(),
+		consumeMetrics: consumeMetrics{
+			handleLatency: opts.InstrumentOptions().MetricsScope().SubScope("consumer").Timer("handle-latency"),
+		},
 	}
 
 	handler := consumer.NewConsumerHandler(s.Consume, opts.ConsumerOptions())
 	return xserver.NewServer(address, handler, opts.ServerOptions()), nil
 }
 
+type consumeMetrics struct {
+	handleLatency tally.Timer
```

Comment:
super nit: Could we call this `handleMessageLatency` with name of `handle-message-latency`?  The full name of `...consumer_handle_latency` makes it sound a little like it's handling an entire consumer when in reality it's just handling a single message as a single consumer.